src/Tools/Analyzer/Phpcpd.php

```diff
@@ -13,12 +13,20 @@ class Phpcpd extends \Edge\QA\Tools\Tool
 
     public function __invoke()
     {
+        $phpcpdNames = array_map(
+            function ($extension) {
+                return "*.{$extension}";
+            },
+            explode(',', $this->config->csv('phpqa.extensions'))
+        );
+
         $args = array(
             'progress' => '',
             $this->options->ignore->bergmann(),
             $this->options->getAnalyzedDirs(' '),
             'min-lines' => $this->config->value('phpcpd.minLines'),
             'min-tokens' => $this->config->value('phpcpd.minTokens'),
+            'names' => $phpcpdNames,
```

Comment:
Please fix [CI](https://travis-ci.org/EdgedesignCZ/phpqa/jobs/620694261#L573).

```suggestion
        );
        $phpcpdNames = array_map(
            function ($extension) {
                return "*.{$extension}";
            },
            array_filter(explode(',', $this->config->csv('phpqa.extensions')))
        );
        if ($phpcpdNames) {
            $args['names'] = \Edge\QA\escapePath(implode(',', $phpcpdNames));
        }
```